src/components/map/mapbox/layers/HeatmapLayer.tsx

```diff
@@ -10,21 +10,21 @@ import MapContext from "../MapContext";
 
 import { Expression, GeoJSONSource, StyleFunction } from "mapbox-gl";
 import {
-  LayersProps,
-  createCenterOfMassDataSource,
   defaultMouseEnterEventHandler,
   defaultMouseLeaveEventHandler,
   findSuitableVisiblePoint,
+  LayersProps,
 } from "./Layers";
 import MapPopup, { PopupType } from "../component/MapPopup";
 import SpatialExtents from "../component/SpatialExtents";
 import SpiderDiagram from "../component/SpiderDiagram";
 import { TestHelper } from "../../../common/test/helper";
-import { FeatureCollection, Point } from "geojson";
+import { Feature, FeatureCollection, Point } from "geojson";
 import { MapDefaultConfig } from "../constants";
+import { generateFeatureCollectiionFrom } from "../../../../utils/GeoJsonUtils";
 import { mergeWithDefaults } from "../../../../utils/ObjectUtils";
 
-interface HeatmapLayer {
+interface IHeatmapLayer {
```

Comment:
since the HeatmapLayer contains a heatmap layer and a cluster layer in it so the config needs contain these two parts. It's an old issue can be refactor with more proper names in the future.

Reply:
Better naming is not included in this ticket, so i just add an `I` in front as a temp solution, to make sure our project doesn't have two objects with the same name.